MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -1042,7 +1043,15 @@ private void initializeModels() {
       R.string.activity_java_services_turf_physical_circle_description,
       new Intent(MainActivity.this, TurfPhysicalCircleActivity.class),
       null,
-      R.string.activity_java_services_tilequery_url, false, BuildConfig.MIN_SDK_VERSION));
+      R.string.activity_java_services_tilequery_url, true, BuildConfig.MIN_SDK_VERSION));
```

Comment:
Same as above, any reason we change the `new` icon flag from `flase` to `true`?

Reply:
Because it's a new example added to the demo app since the last Play Store release.